addons/calendar/views/calendar_views.xml

```diff
@@ -362,7 +362,6 @@
                 <field name="attendees_count" invisible="1"/>
                 <field name="accepted_count" invisible="1"/>
                 <field name="declined_count" invisible="1"/>
-                <field name="user_can_edit" invisible="1"/>
```

Comment:
This could break custom code relying on this field presence in the view (attrs or xpaths for instance).

I suggest you keep it on the stable release.